com.unity.render-pipelines.core/Runtime/Debugging/DebugManager.cs

```diff
@@ -243,6 +258,25 @@ void OnPanelDirty(DebugUI.Panel panel)
             onSetDirty();
         }
 
+        /// <summary>
+        /// Request DebugWindow to open the specified panel.
+        /// </summary>
+        /// <param name="index">Index of the debug window panel to activate.</param>
+        public void RequestEditorWindowPanelIndex(int index)
+        {
+            // Similar to RefreshEditor(), this function is required to bypass a dependency problem where DebugWindow
+            // cannot be accessed from the Core.Runtime assembly. Should there be a better way to allow editor-dependent
+            // features in DebugUI?
+            m_RequestedPanelIndex = index;
+        }
+
+        internal int? GetRequestedEditorWindowPanelIndex()
+        {
+            int? requestedIndex = m_RequestedPanelIndex;
+            m_RequestedPanelIndex = null;
+            return requestedIndex;
+        }
+
         // TODO: Optimally we should use a query path here instead of a display name
```

Comment:
Can we remove this TODO and create a task in Jira for this?

Reply:
I don't know what this is referring to, @JulienIgnace-Unity do you happen to have a jira for this?